game/src/main/java/org/apollo/game/fs/decoder/SynchronousDecoder.java

```diff
@@ -1,16 +1,12 @@
 package org.apollo.game.fs.decoder;
 
+import org.apollo.util.ThreadUtil;
+
 import java.util.Arrays;
 import java.util.List;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
You've ran IntelliJ's "optimize imports" function over the code, this isn't the place to be doing that.

Reply:
I have it set to auto do it. Sorry.